dali/test/python/test_operator_audio_resample.py

```diff
@@ -34,64 +34,73 @@
 rates = [ 16000, 22050, 12347 ]
 lengths = [ 10000, 54321, 12345 ]
 
-def create_test_files():
+def create_files():
```

Comment:
I think this would be another way, more verbose probably:

```suggestion
from nose.tools import nottest
@nottest
def create_files():
```